src/utilities/activity-utils.ts

```diff
@@ -126,9 +124,9 @@ export const enableReportButton = (activity: Activity) => {
   return !hasCompletionPage && activity.student_report_enabled;
 };
 
-export const getLinkedPluginEmbeddable = (page: Page, id: string) => {
-  const linkedPluginEmbeddable = page.embeddables.find((e: EmbeddableWrapper) => e.embeddable.embeddable_ref_id === id);
-  return linkedPluginEmbeddable?.embeddable.type === "Embeddable::EmbeddablePlugin" ? linkedPluginEmbeddable.embeddable : undefined;
+export const getLinkedPluginEmbeddable = (section: SectionType, id: string) => {
+  const linkedPluginEmbeddable = section.embeddables.find((e: EmbeddableType) => e.embeddable_ref_id === id);
```

Comment:
I'm not sure that LARA is going to add these plugins to the same section as the embeddable they are wrapping. And if the embeddable is moved to another section I'd guess that LARA is not going to move the plugins at the same time.

Because of this it would be safer if this code would search the whole page not just the current section.

Reply:
Fixed this to search through the page instead.

Reply:
looks good.